### DISCUSSION TITLE
Token Refresh [Aug 12]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3256] [Discord Threads]
> 🙋‍♂️ **ccoli**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Tried this?   
#### https://docs.nocodb.com/developer-resources/rest-apis#auth-apis   
####    
####     /api/v1/auth/token/refresh  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1007169305177243730)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
